From 8bdb8b8b75b440394e6f70b8cbb50474cd608404 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 8 Nov 2020 21:37:08 +0200
Subject: [PATCH 11/15] lx2160a: fix SVR 12c mask

Bit 12 must be masked when looking at SVR.
Issue was found when testing 12 core variant.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 plat/nxp/soc-lx2160/include/soc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/soc-lx2160/include/soc.h b/plat/nxp/soc-lx2160/include/soc.h
index f4e5e0dcd326..f9c270be2f1b 100644
--- a/plat/nxp/soc-lx2160/include/soc.h
+++ b/plat/nxp/soc-lx2160/include/soc.h
@@ -123,7 +123,7 @@
 
 /* bits */
 /* SVR Definition */
-#define SVR_WO_E		0xFFFFFE
+#define SVR_WO_E		0xFFFFEE
 #define SVR_LX2160A		0x873601
 #define SVR_LX2120A		0x873621
 #define SVR_LX2080A		0x873603
-- 
2.30.2

